src/js/tour.js

```diff
@@ -148,6 +148,17 @@ export class Tour extends Evented {
     return this.currentStep;
   }
 
+  /**
+   * Hide the current step
+   */
+  hide() {
+    const currentStep = this.getCurrentStep();
+
+    if (currentStep && typeof currentStep.hide === 'function') {
```

Comment:
We should use `isFunction` from lodash. We're using a few lodash functions throughout the code. Webpack tree shakes things for us, so if you just import the function you need our build stays small. Also, is there a case when then step would not have a `hide` function on it? If not, we could probably remove this check.

Reply:
@BrianSipple if you can update or remove this, we'll be good to go 👍

Reply:
Agreed that the function check probably shouldn't be necessary. Updated by removing it.